.eslintrc

```diff
@@ -4,7 +4,8 @@
     "dist",
     "public",
     "node_modules",
-    "packages/schema/schema"
+    "packages/schema/schema",
+    "packages/kit"
```

Comment:
Can we revert the linting for now and just include the JSdoc + readability improvements? 🙏